packages/toolkit/src/helpers/useEnvironment.js

```diff
@@ -0,0 +1,177 @@
+import Web3 from 'web3'
```

Comment:
Here we handle the environment logic initialization. We setup web3, apm and ipfs.